components/tailwind.config.base.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
       info: "var(--color-info-500)",
       primary: {
         300: "var(--color-primary-300)",
+        500: "var(--color-primary-500)",
```

Comment:
@danielleflinn reports uses 300, tabs seems to use 500. What do you think about standardizing on one variant?